Gems/AtomLyIntegration/AtomFont/Code/Include/AtomLyIntegration/AtomFont/AtomFont.h

```diff
@@ -16,6 +16,7 @@
 #include <AzCore/std/containers/map.h>
 #include <AzCore/std/smart_ptr/weak_ptr.h>
 #include <AzCore/std/parallel/shared_mutex.h>
+#include <AzCore/Asset/AssetCommon.h>
 #include <map>
```

Comment:
Unrelated to your change, but is this header necessary? Seems like we don't use STL map unlike the AZStd::map.

Reply:
There is an stl map

        using FontMap = std::unordered_map<AzFramework::FontId, FFont*>;
        
Now, why is there a mixture of std::unordered_map and AZStd::unordered_map in the same class/file is a mystery to me. Probably just an oversight while porting some legacy code :/

Reply:
That is likely an oversight, and should have been converted to AZStd::map when I converted CryFont to AtomFont.